vendor/engines/bulk_email/app/models/bulk_email/delivery_form.rb

```diff
@@ -41,6 +46,14 @@ def recipients
       @recipients ||= User.where_ids_in(recipient_ids)
     end
 
+    def record_delivery
+      Job.create!(
```

Comment:
It might be worth prefixing this with `BulkEmail::` just to make sure it's clear which type of `Job` this is.